tests/install_upgrade_operators/golden_images/constants.py

```
@@ -0,0 +1,45 @@
+from tests.install_upgrade_operators.constants import CUSTOM_DATASOURCE_NAME
+from utilities.constants import AMD_64, ARM_64, OUTDATED, S390X, WILDCARD_CRON_EXPRESSION
+
+HCO_CR_DATA_IMPORT_SCHEDULE_KEY = "dataImportSchedule"
+RE_NAMED_GROUP_MINUTES = "minutes"
+RE_NAMED_GROUP_HOURS = "hours"
+DATA_IMPORT_SCHEDULE_RANDOM_MINUTES_REGEX = (
+    rf"(?P<{RE_NAMED_GROUP_MINUTES}>\d+)\s+" rf"(?P<{RE_NAMED_GROUP_HOURS}>\d+)\/12\s+\*\s+\*\s+\*\s*$"
```

Ruff (0.14.11)

Implicitly concatenated string literals on one line

Combine string literals

(ISC001)